src/tasks/copy-template-files.ts

```diff
@@ -213,7 +217,32 @@ const processTemplatedFiles = async (
         templateFileDescriptor.relativePath.split(templateTargetName)[0],
         templateTargetName,
       );
-      fs.writeFileSync(targetPath, output);
+
+      let finalOutput = output;
+
+      const containsNextjs = targetPath.includes("nextjs");
+      const containsDotfile = /\/\.[^/]+/.test(targetPath);
+      const hardhatOrFoundry = targetPath.split("packages")[1]?.split("/")[1];
+      const isHardhatOrFoundry =
+        hardhatOrFoundry && (hardhatOrFoundry.includes("hardhat") || hardhatOrFoundry.includes("foundry"));
```

Comment:
and here `===` instead of `.includes` since in the future it can be folders like `foundry-addon`

Reply:
Ohh yes! previously I was working with array so used .includes but then refactored the logic a bit and forgot to update it here 

updated at https://github.com/scaffold-eth/create-eth/pull/55/commits/9712cc3b8ab4ff978c88284618ab3bb3e69dac44, Thanks !!